core/src/test/scala/caliban/TestUtils.scala

```diff
@@ -393,12 +393,13 @@ object TestUtils {
       )
 
       @GQLInterface
-      sealed trait WithNullable {
```

Comment:
This interface was never generated previously since there weren't multiple case classes that shared any fields.

Reply:
Doesn't it create an interface even if there is only one extending class?

Reply:
It did, it was the type in the resolver that was wrong.